robottelo/cleanup.py

```diff
@@ -0,0 +1,20 @@
+# -*- encoding: utf-8 -*-
+"""Cleanup module for different entities"""
+from robottelo.cli.location import Location
+from robottelo.cli.org import Org
+from robottelo.cli.proxy import Proxy
+
+
+def capsule_cleanup(proxy_id=None):
+    """Deletes the capsule with the given id"""
+    Proxy.delete({'id': proxy_id})
```

Comment:
No input validation done and this is the responsibility of the caller at this time.  This can be improved later, if necessary.  The same is applicable to all functions in this module.